.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# see:
+# https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners
+* @openziti/maintainers
```

Comment:
whooo hoo!

Reply:
maintainers still need to be poked to review PRs